app/services/parser.rb

```diff
@@ -0,0 +1,64 @@
+require 'open-uri'
+
+class Parser
+  HEADLESS_RUN = true
+  DELAY_VAL = 3
+
+  def start(count=2)
+    if HEADLESS_RUN
+      headless = Headless.new
+      headless.start
+    end
+    
+    doc = Nokogiri::HTML(open('https://www.avito.ru/irkutsk/kvartiry/sdam/na_dlitelnyy_srok'));
```

Comment:
Do not use semicolons to terminate expressions.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.